ts/data/case.ts

```diff
@@ -649,7 +660,7 @@ function addTooltipRow(
   container: HTMLElement, // must be a two column grid
   label: string,
   name: string,
-  misoUrl: string,
+  misoUrl?: string,
```

Comment:
Why would miso url be optional?

Reply:
This function is for adding a row (basically a key and value pair) to the tooltip. Previously included rows all had MISO links, but this PR adds the assay without a MISO link